acvm-repo/acvm/src/pwg/mod.rs

```diff
@@ -187,7 +164,7 @@ pub enum OpcodeNotSolvable<F> {
     ExpressionHasTooManyUnknowns(Expression<F>),
 }
 
-/// Allows to point to a specific opcode as cause in errors.
+/// Allowed to point to a specific opcode as an error's cause.
```

Comment:
The tense of the previous comment looks to more be describing what this enum enables. This re-wording makes me think what is `ErrorLocation` "allowed" to do. If we don't like the word "allows" we could change to something like "Used by errors to point to a specific opcode as that error's cause".